modules/reactor/doc/content/source/meshgenerators/PatternedHexMeshGenerator.md

```diff
@@ -38,6 +38,9 @@ In addition, [!param](/Mesh/PatternedHexMeshGenerator/assign_control_drum_id) ca
 
 These `MeshMetaData` as well as `control_drum_id` can be used by other MOOSE objects such as [`MultiControlDrumFunction`](/MultiControlDrumFunction.md) to simulate control drums rotation during power transients.
 
+## Interface Boundaries
```

Comment:
if these paragraphs are all the same then you could include them with shortcuts rather than duplicate

Reply:
maybe just the part that is in common since the parameters are different

Reply:
Sure